Update VMO prereq (#1506)
spectrocloud/librarium: content/docs/04.6-vm-management.md

- Palette version 3.3.0 or higher.

- For data centers, production VMs are supported on bare metal Kubernetes clusters deployed on Canonical MAAS. To learn how to configure MAAS and create MAAS clusters in Palette, refer to the [Install and Manage MAAS Gateway](/clusters/data-center/maas/install-manage-maas-pcg) guide.

> - For VMO on Edge, please contact our support team by sending an email to support@spectrocloud.com.

Reviewer: ```suggestion
-  Contact our support team to learn more about VMO on Edge and how you can get started.
```
